lib/theme_check/checks/default_locale.rb

```diff
@@ -7,7 +7,9 @@ class DefaultLocale < JsonCheck
 
     def on_end
       return if @theme.default_locale_json
-      add_offense("Default translation file not found (for example locales/en.default.json)")
+      add_offense("Default translation file not found (for example locales/en.default.json)", line_number: -1) do |corrector|
```

Comment:
Does it work if you simply do not pass the `line_number`?

```suggestion
      add_offense("Default translation file not found (for example locales/en.default.json)") do |corrector|
```

Reply:
No it doesn't. I decided to modify the `correctable?` method in `offense.rb `so it doesn't rely on `line_number`